addon/utils/normalize-auto-complete.js

```diff
@@ -0,0 +1,26 @@
+import { computed } from '@ember/object';
+
+/**
+ * Normalizes a boolean as a HTML auto-complete value
+ *
+ * @param {Boolean} autoComplete
+ */
+export function normalizeAutoComplete(autoComplete) {
+  if (autoComplete == null) {
```

Comment:
This double-equals doesn't cause a linter complaint? 😮

Reply:
seems tests are passing...so guess no